test/tests.js

```diff
@@ -18,6 +18,7 @@ import "./main-hook-tests.mjs"
 import "./require-hook-tests.js"
 import "./repl-hook-tests.mjs"
 import "./scenario-tests.mjs"
+import "./_external/test262/index.js"
 
```

Comment:
We call the external folder in `src` the `vendor` folder. We could call it that in test too.

Reply:
will do.